A32NX/html_ui/Pages/VCockpit/Instruments/Airliners/A320_Neo/SAI/A320_Neo_SAI.js

```diff
@@ -1148,8 +1233,188 @@ class A320_Neo_SAI_SelfTestTimer extends HTMLElement {
+    connectedCallback() {
+        this.construct();
+    }
+
+    construct() {
+        Utils.RemoveAllChildren(this);
+
+        this.complete = true;
+
+        this.start_time = 10;
+        this.resetTimer = this.start_time;
+
+        const att_x = 33.5;
+        const boxHeight = 7;
+        const boxWidth = 30;
+        const boxRow = 64;
+        const txt_off_x = 1.5;
+        const txt_off_y = 6;
+
+        this.attResetDiv = document.createElement("div");
+        this.attResetDiv.id = "AttResetDiv";
+        this.attResetDiv.setAttribute("border", "none");
+        this.attResetDiv.setAttribute("position", "absolute");
+        this.attResetDiv.setAttribute("display", "block");
+        this.attResetDiv.setAttribute("top", "0%");
+        this.attResetDiv.setAttribute("left", "0%");
+        this.attResetDiv.setAttribute("width", "100%");
+        this.attResetDiv.setAttribute("height", "100%");
+        this.attResetDiv.style.display = "none";
+        this.appendChild(this.attResetDiv);
+
+        this.attResetSVG = document.createElementNS(Avionics.SVG.NS, "svg");
+        this.attResetSVG.setAttribute("id", "AttResetSVG");
+        this.attResetSVG.setAttribute("viewBox", "0 0 600 600");
+        this.attResetSVG.style.position = "absolute";
+        this.attResetSVG.style.top = "0%";
+        this.attResetSVG.style.left = "0%";
+        this.attResetSVG.style.width = "100%";
+        this.attResetSVG.style.height = "100%";
+        this.attResetDiv.appendChild(this.attResetSVG);
+
+        const st_att = document.createElementNS(Avionics.SVG.NS, "rect");
+        st_att.setAttribute("id", "AttReset")
+        st_att.setAttribute("fill", "#afbb3a");
+        st_att.setAttribute("x", att_x + "%");
+        st_att.setAttribute("y", boxRow + "%");
+        st_att.setAttribute("width", boxWidth + "%");
+        st_att.setAttribute("height", boxHeight + "%");
+        this.attResetSVG.appendChild(st_att);
+
+        this.st_att_txt = document.createElementNS(Avionics.SVG.NS, "text");
+        this.st_att_txt.setAttribute("id", "AttResetTxt")
+        this.st_att_txt.textContent = "ATT "+ Math.ceil(this.resetTimer)  +"s";
+        this.st_att_txt.setAttribute("font", "Roboto");
+        this.st_att_txt.setAttribute("font-weight", "900");
+        this.st_att_txt.setAttribute("font-size", "42px");
+        this.st_att_txt.setAttribute("fill", "#1f242d");
+        this.st_att_txt.setAttribute("x", att_x + txt_off_x + "%");
+        this.st_att_txt.setAttribute("y", boxRow + txt_off_y + "%");
+        this.attResetSVG.appendChild(this.st_att_txt);
+
+    }
+
+    update(dTime) {
+        if (this.complete) return;
+        if (this.resetTimer >= 0) {
+            this.resetTimer -= dTime / 1000;
+        }
+        if (this.resetTimer <= 0) {
+            this.finishReset();
+        }
+        if (this.resetTimer > 9) {
+            this.st_att_txt.textContent = "ATT " + Math.ceil(this.resetTimer) + "s";
+        }
+        else {
```

Comment:
Don't spread an if else on multiple lines, use `} else {`

Reply:
Every if else (in this file at least) is formatted in this way, should they all be changed?

Reply:
you're right 2hwk to match the surrounding code

Reply:
resolved with linting